internal/driver/driver.go

```diff
@@ -727,10 +703,33 @@ func (d *Driver) RefreshAllDevicePaths() {
 	}
 }
 
+// Deprecated: 3.0 version of the service supports Path which is a string instead of Paths which is []string which leads to breaking change.
+// updatePathToPaths takes care of this breaking change.
+func (d *Driver) updatePathToPaths(cd models.Device) {
+	d.mutex.Lock()
+	defer d.mutex.Unlock()
+
+	if value, ok := cd.Protocols[UsbProtocol][Path]; ok {
+		if value != nil {
+			cd.Protocols[UsbProtocol][Paths] = []string{value.(string)}
+			delete(cd.Protocols[UsbProtocol], "Path")
```

Comment:
I thought we had discussed keeping both fields in the metadata because otherwise, when you get a device it will be missing the `Path` field that used to be there. @lenny-intel thoughts?

Reply:
I don't remember discussing about this, you mean creating `Metadata` section within protocols like how we have in onvif service? Existing `Path` field will be changed to `Paths` with this code which is easier to maintain. We have added in the docs too that `Path` field is deprecated and users should be using `Paths`. I think @lenny-intel was fine with this.

Reply:
I mean keeping bath `Path` and `Paths` in the `Onvif` protocol properties, and using code to sync them. As long as @lenny-intel is OK with this approach I am fine as well, since this will be cleaner.

Reply:
Yes, we have to keep both. My thought is if a device is loaded that has Path set, then set Paths to the single Path. That way the driver code always deals with Paths.